doc-Monitoring_Alerts_and_Reporting/topics/To_schedule_a_Report.adoc

```diff
@@ -34,29 +45,26 @@ image:2279.png[]
 * Type or select a date to begin the schedule in *Starting Date*.
 * Select a *Starting Time* based on a 24 hour clock in the {product-title} appliance's Time Zone.
 
-. To send an email that includes an attachment with the report contents, check *Send an E-mail*. Parameters required for sending an email are displayed.
-+
-
-image:2281.png[]
-
+. Use the *E-Mail after Running* feature to send an email after the report has been generated to selected users. The email will include a link to the report.
+* check *Send an E-mail*. 
 * In *From (leave blank for default)*, type in the sending email.
 * Use *Add a User*, to select a specific user.
   The user must have a valid email address entered under accounts.
 * Use *Add (enter manually)* to type in the address not registered to a User.
 * Then, click  image:1862.png[] *(Add)*.
 
 . If you are sending an email after the report runs, then you can select further options under Email Options.
-+
 
-image:2283.png[]
-+
 * Check *Send if Report is Empty* if you want an email even if no records exist in the report.
 * Next to Attachments, check if you would like the report attached as a *Text*, *CSV*, or *PDF* file.
 
-. Click *Add*.
+. Click *Add*
+
+NOTE: See the [General Configuration](https://access.redhat.com/documentation/en/red-hat-cloudforms/4.2-beta/paged/general-configuration/) guide, to learn how to verify the address and validate outgoing email settings.
```

Comment:
This link is showing up a bit odd for me in the preview. Would you mind checking it?
It might just be the order of the brackets/components -- perhaps try a format like:
 See https://access.redhat.com/documentation/en/red-hat-cloudforms/version-4.0/deployment-planning-guide/#cpu_sizing_assistant_for_a_dedicated_vmdb_html[CPU Sizing Assistant for a Dedicated VMDB Host] in the _Deployment Planning Guide_